etna/transforms/embeddings/models/ts2vec.py

```diff
@@ -99,18 +84,46 @@ def __init__(
             batch_size=self.batch_size,
         )
 
-        self._is_fitted: bool = False
+        self.__is_freezed: bool = False
+
+    @property
+    def is_freezed(self):
+        """Return whether to skip fit of a pretrained model."""
+        return self.__is_freezed
+
+    def freeze(self, is_freezed: bool = False):
```

Comment:
I think the `default` value should be `True`. It doesn't make sense to call `model.freeze()`, but it isn't freezed, but unfreezed.